extensions/amp-anim/0.1/amp-anim.js

```diff
@@ -105,7 +105,13 @@ class AmpAnim extends AMP.BaseElement {
       return Promise.resolve();
     }
     this.img_.setAttribute('src', src);
-    this.loadPromise_ = loadPromise(this.img_);
+    this.loadPromise_ = loadPromise(this.img_)
+        .catch(error => {
+          if (this.img_.getAttribute('src')) {
```

Comment:
yep, sorry just made the change hastily here and just havent pushed out latest changes with tests.
will mark done when ive pushed.